### PR TITLE
Add breakdown for aura effect for aura skills

### PR DESCRIPTION
**Pull request by chinhodado**
image

### CODE REVIEW

File: Modules/CalcOffence-3_0.lua [hunk 1]
```diff
@@ -388,6 +388,14 @@ function calcs.offence(env, actor, activeSkill)
 			breakdown.AreaOfEffectMod = breakdown.mod(skillCfg, "AreaOfEffect")
 		end
 	end
+
```
Comment (Quotae) [inline Modules/CalcOffence-3_0.lua]: I'd say remove these extra blank gaps between this if statement and the ones above and below it. They're unnecessary besides looking better when you're editing this file, and fewer lines in the file is better.

File: Modules/CalcSections-3_0.lua [hunk 2]
```diff
@@ -475,6 +475,10 @@ return {
 		{ breakdown = "AreaOfEffectMod" }, 
 		{ modName = "AreaOfEffect", cfg = "skill" },
 	}, },
+	{ label = "Aura Effect", haveOutput = "AuraEffectMod", { format = "x {2:output:AuraEffectMod}",
```
Comment (Quotae) [inline Modules/CalcSections-3_0.lua]: Change the label to "Aura Effect Mod", as that's consistent with the other mods that show up in this box.

Also, move it in the list. I'd put it below the duration mods, above the projectile mods. In this current spot, it splits up the AoE Mod and the Radius display, which just looks weird.

Comment (chinhodado) [reply Modules/CalcSections-3_0.lua]: Sure, done

File: Modules/CalcOffence-3_0.lua (hunk 1 above)
Comment (chinhodado) [reply Modules/CalcOffence-3_0.lua]: I think it makes it easier to read, and 2 blank lines isn't really a big deal when the file is already 2000+ lines long. But if people feel strongly about it I'll remove not, it's not a big deal.

**Pull request merged**